mojoproject.toml

```diff
@@ -7,23 +7,25 @@ platforms = ["osx-arm64", "linux-aarch64", "linux-64"]
 version = "0.1.0"
 
 [tasks]
-filter-single-image = { cmd="mojo package operations/ -o operations.mojopkg && python examples/filter_single_image.py", env={ MODULAR_ONLY_USE_NEW_EXTENSIBILITY_API="true" }}
+package_ops = { cmd="mojo package operations/ -o operations.mojopkg", env={ MODULAR_ONLY_USE_NEW_EXTENSIBILITY_API="true" } }
```

Comment:
One trick that Alex showed me is to use `depends-on` with a task to do the Mojo custom op builds for each Python task, like we do here: https://github.com/modular/max/blob/main/examples/custom_ops/mojoproject.toml#L10 . Also, thankfully we can now totally drop the `MODULAR_ONLY_USE_NEW_EXTENSIBILITY_API`, we moved over everything to the new API in mid-January.